packages/manager/modules/vrack/src/vrack.controller.js

```diff
@@ -214,6 +226,14 @@ export default class VrackMoveDialogCtrl {
       });
   }
 
+  static isServiceAllowed(service) {
+    return (
+      !service.expired &&
+      [STATUS.ok, STATUS.delivered].includes(service.state) &&
+      service.status === STATUS.ok
```

Comment:
may be you can remove this condition "service.status === STATUS.ok". The above condition "[STATUS.ok, STATUS.delivered].includes(service.state)" covers this.

Reply:
@radireddy hmm no, `status` (which comes from `/serviceInfos`) can be different from `state`

Reply:
ok got it. I was thinking both state and status are the same.